tokio/src/net/tcp/mod.rs

```diff
@@ -7,7 +7,7 @@
 //! [`connect`] method, which returns a future which returns a `TcpStream`.
 //!
 //! To listen on an address [`TcpListener`] can be used. `TcpListener`'s
-//! [`incoming`][incoming_method] method can be used to accept new connections.
+//! [`incoming`][TcpListener::incoming] method can be used to accept new connections.
```

Comment:
Nit: can we put this reference at the end of the doc comment block like we do for other links?